ait/gui/static/js/ait/gui/Search.js

```diff
@@ -139,8 +139,12 @@ const MnemonicSearch = {
         )
 
         if (curPacket !== null) {
-            val = curPacket.__get__(this._selection)
-            raw = curPacket.__get__(this._selection, true)
+            if ( this._selection in curPacket['dntoeu']) {
```

Comment:
Since we're probably going to be doing a lot of this "grab DN to EU if it exists otherwise grab raw value" thing we should consider wrapping this behavior in a general `get` helper function of some sort.

Reply:
You mean as a utility method (for use outside of just Search)?